circle.yml

```diff
@@ -539,7 +539,7 @@ jobs:
       - run:
           name: Package
           # The package must be run within the actual crate and not in the workspace.
-          working_dir: bindings/rust
+          working_dir: ~/bindings/rust
```

Comment:
```suggestion
          working_dir: ~/project/bindings/rust
```
if absolute path is required.

Reply:
Ok, it should have been `working_directory`.

Reply:
Neither `bindings/rust`, `~/bindings/rust`, or `~/project/bindings/rust` works. Will check `pwd` because this is confusing.